docker-compose.yml

```diff
@@ -131,6 +131,8 @@ services:
       context: ./
     ports:
       - "53589:53589"
+    env_file:
+      - .env
```

Comment:
I might be mistaken, but the `.env` file is handled as a default; so I don't believe it needs to be specified explicitly (https://docs.docker.com/compose/environment-variables/#the-env-file)

Reply:
It was necessary because we needed to add `CERTIFICATE_DB` to be read for [your taskd](https://github.com/coddingtonbear/taskserver/blob/inthe_am_dockerized/src/daemon.cpp#L591) modified daemon. But if we use the default paths not it's not (see [my previous comment](https://github.com/coddingtonbear/inthe.am/pull/362#discussion_r667353870))